assets/javascripts/card-form-customization.js

```diff
@@ -58,6 +58,28 @@ function getDesignFormValues() {
   return formValues;
 }
 
+function handleFontChange() {
+  const fontName = document.getElementById('custom_sf_font_name');
+
+  if (fontName.value === OMISE_CUSTOM_FONT_OTHER) {
+    document.getElementById('custom_sf_custom_font_name').style.display = null
```

Comment:
Why not `fontName.style.display`?

Reply:
These are two different elements. One has id of `custom_sf_font_name` and another has `custom_sf_custom_font_name`

Reply:
oops 🤦🏽